### PR TITLE
Initial commit of r-flowml to conda-forge

### PR DESCRIPTION
**Pull request by apeltzer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This was created using the r-conda skeleton helper script. No further modifications were made.

### CODE REVIEW

**Comment by SMLMS** [comment]
I agree to be listed here.

**Comment by apeltzer** [comment]
Needs https://github.com/conda-forge/staged-recipes/pull/24273 and https://github.com/conda-forge/staged-recipes/pull/24274

@neslihanky  / @SMLMS / @KaBrrrp please also comment on both these issues that you're ok in helping maintain these - thanks 👍🏻

**Comment by KaBrrrp** [comment]
I agree to be listed here.

**Comment by apeltzer** [comment]
@conda-forge/help-r ready for review 👍🏻  This needs the packages #24274 and #24273 first to be merged to have all dependencies available 👍🏻

**Comment by apeltzer** [comment]
Lets wait for 1-2 hours and then restart the CI, which should then find all the required packages 🥳

File: recipes/r-flowml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    - r-vip
+
+test:
+  commands:
+    - $R -e "library('flowml')"           # [not win]
+    - "\"%R%\" -e \"library('flowml')\""  # [win]
+
+about:
+  home: https://github.com/Boehringer-Ingelheim/flowml
+  license: GPL-3
```
Comment (cbrueffer) [inline recipes/r-flowml/meta.yaml:77]:
```suggestion
  license: GPL-3.0-or-later
```

**Review by cbrueffer — CHANGES_REQUESTED** [review]
One small change. As you said, best to apply once the dependencies are available.

File: recipes/r-flowml/meta.yaml (hunk 1 above)
Comment (apeltzer) [reply recipes/r-flowml/meta.yaml:77]: I will do so, once I see the r-abcanalysis package available on the package index. The r-fastshap one is there now 🥳

**Comment by apeltzer** [comment]
If I try locally, the packages where available to mamba, so I pushed commit. CI here seems to be not to be able to find the required packages. Maybe wait and restart CI?

**Review by cbrueffer — APPROVED** [review]

**Pull request merged**